controllers/runnermanager.go

```diff
@@ -263,6 +286,41 @@ func difference(prev, current []string) []string {
 	return ret
 }
 
+func (m *managerLoop) notifyToSlack(ctx context.Context, runnerList []*github.Runner, podList *corev1.PodList) error {
+	lastCheckTime := m.lastCheckTime.Load().(time.Time)
+
+	for i := range podList.Items {
+		po := &podList.Items[i]
+		jobResult, err := m.runnerPodClient.GetJobResult(ctx, po.Status.PodIP)
+		if err != nil {
+			m.log.Error(err, "skipped notification because failed to get the job result from the runner pod API", "pod", namespacedName(po.Namespace, po.Name))
+			continue
+		}
+
+		if jobResult.Status == rc.JobResultUnfinished {
+			m.log.Info("skipped notification because pod is not finished", "pod", namespacedName(po.Namespace, po.Name))
+			continue
+		}
+		if jobResult.FinishedAt.Before(lastCheckTime) {
+			m.log.Info("skipped notification because pod status is not updated", "pod", namespacedName(po.Namespace, po.Name))
+			continue
+		}
+
+		if len(m.slackAgentServiceName) != 0 {
+			fmt.Println("Send an notification to slack jobResult = ", jobResult)
+			c, err := agent.NewClient(fmt.Sprintf("http://%s", m.slackAgentServiceName))
+			if err != nil {
+				return err
+			}
+			return c.PostResult(ctx, m.slackChannel, jobResult.Status, *jobResult.Extend, po.Namespace, po.Name, jobResult.JobInfo)
+		} else {
+			fmt.Println("Skip sending an notification to slack because Slack agent service name is blank")
```

Comment:
```suggestion
			m.log.Info("skip sending a notification to slack-agent because service name is blank", "pod", namespacedName(po.Namespace, po.Name))
```